### ISSUE TITLE
"Current workspace is not a godot project"

### ISSUE THREAD

**Issue by 11clock** [GodotExplorer/godot-tools#51]
I configured the editor path to the directory my executable is located in, but I am still getting "Current workspace is not a godot project" error when I attempt to run the project from VS Code.

My user settings look like the below:

```
{
    "editor.codeLens": false,
    "csproj.itemType": {
        "*": "Content",
        ".ts": "TypeScriptCompile",
        ".cs": "Compile"
    },
    "csharp.format.enable": false,
    "GodotTools.completeNodePath": true,
    "GodotTools.editorPath": "C:\\Godot_v3.0-stable_win64.exe\\"
}
```

**Comment by Geequlim** [comment]
What is the godot version of your project?

**Comment by 11clock** [comment]
I am using Godot 3.0.

**Comment by Geequlim** [comment]
Fixed by ef3a70f4173811af94d43a49b173fa7b1bf432d6
